src/Pkg.jl

```diff
@@ -97,6 +97,18 @@ See also [`PackageSpec`](@ref).
 """
 const add = API.add
 
+"""
+    Pkg.precompile()
+
+Precompile all the dependencies of the project by running `import` on all of them in a new process.
+
+# Examples
+```julia
+Pkg.precompile() # precompiles all the packages in this project.
```

Comment:
Don't think this is needed since it is included in just above:
```suggestion
Pkg.precompile()
```